documentation/en/source/pages/templates/templates-spec/overview.rst

```diff
@@ -16,4 +16,4 @@ A template has two main parts:
    :titlesonly:
 
    stack
-   builders/overview
\ No newline at end of file
+   ../../machine-images/builders/overview
```

Comment:
does this build correctly? I think the reference should be in the intro:
builders: an array defining the format of the machine images to build. For more information on builders, refer to :ref:`template-builders`
Same info should be added to the new section "Migration Specifications"